styles/styles.css

```diff
@@ -782,3 +782,125 @@ main .section.small-vertical-margins {
         text-align: left;
     }
 }
+
+.getting-started main .default-content-wrapper ul {
+    margin: 0;
+    padding: 10px;
+}
+
+.getting-started main .default-content-wrapper li {
+    padding-bottom: 10px;
+}
+
+.getting-started main .default-content-wrapper p {
+    margin: 0 0 10px;
+}
+
+.getting-started main .default-content-wrapper img {
+    transform: scale(0.6);
+}
+
+main .section[data-background] picture.section-background-img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    object-fit: cover;
+    z-index: -2;
+    overflow: hidden;
+}
+
+main .section[data-background] picture.section-background-img img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.getting-started main .default-content-wrapper p > strong:first-child:last-child {
+    font-size: 1.7rem;
```

Comment:
on desktop the font size should be `20px`

Reply:
we should use variables for the font-size

Reply:
> we should use variables for the font-size

agree but needs some general review. so not for this PR I think https://github.com/hlxsites/bevespi/issues/136